BitmarkRegistry/Controllers/Onboarding/LoginViewController.swift

```diff
@@ -41,21 +51,40 @@ class LoginViewController: BaseRecoveryPhraseViewController {
   }
 
   // MARK: - Handlers
+  @objc func changePhraseOptions(_ sender: UIButton) {
+    if numberOfPhrases == 12 {
+      changePhraseOptionsButton.setTitle(changeTo12phrases, for: .normal)
+      numberOfPhrases = 24
+    } else {
+      changePhraseOptionsButton.setTitle(changeTo24phrases, for: .normal)
+      numberOfPhrases = 12
+    }
+    _numericOrders = nil
+    clearForm()
+    recoveryPhraseCollectionView.reloadData()
+  }
+
   @objc func tapToSubmit(_ sender: UIButton) {
     do {
-      let account = try AccountService.getAccount(phrases: getUserInputPhrases())
+      let userInputPhrases = getUserInputPhrases()
+      guard let seedVersion = SeedVersion(numberOfPhrases: userInputPhrases.count) else {
```

Comment:
I suggest to put throw in init function.

Reply:
that's ok. but as I research in docs.swift that init? is Failable Initializer
```To cope with initialization conditions that can fail, define one or more failable initializers as part of a class, structure, or enumeration definition. You write a failable initializer by placing a question mark after the init keyword (init?).```
Reference: https://docs.swift.org/swift-book/LanguageGuide/Initialization.html#//apple_ref/doc/uid/TP40014097-CH18-XID_339 - Failable Initializers
Thanks

Reply:
I don't find any strong clue to state that we should not use throwing initialization. Most likely it's to be compatible with Objective-C version, which does not have throw feature. I'm ok with this approach.